gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1446,6 +1446,21 @@ protected void configureConversionUtilities(Project project, SourceSet sourceSet
         project.getLogger().lifecycle("You can use '-PconvertToPdl.reverse=true|false' to change the direction of conversion.");
       }));
     });
+
+    project.getTasks().create(sourceSet.getTaskName("reformat", "Pdl"), TranslateSchemasTask.class, task ->
```

Comment:
do we plan to run this task for migration already done? does it need to run for new migrations?